src/appmixer/utils/storage/GetMany/GetMany.js

```
@@ -0,0 +1,36 @@
+                results.push(value);
+            }
+        }
+        // Send results to 'out' if any results are found
+        if (results.length > 0) {
+            context.sendJson({ results }, 'out');
+        }
+
+        // Send notFound to 'notFound' if any are found
+        if (notFound.length > 0) {
+            context.sendJson({ notFound }, 'notFound');
+        }
```

ESLint

[error] Closing curly brace does not appear on the same line as the subsequent block.

(brace-style)